src/commands.c

```diff
@@ -2240,10 +2242,23 @@ void cmd_gaps(I3_CMD, const char *type, const char *scope, const char *mode, con
     }
 
     if (!strcmp(type, "inner")) {
-        CMD_GAPS(inner, outer);
-    } else if (!strcmp(type, "outer")) {
-        CMD_GAPS(outer, inner);
+        CMD_GAPS(inner);
     } else {
+        if (!strcmp(type, "top") || !strcmp(type, "vertical") || !strcmp(type, "outer")) {
```

Comment:
I think we can just use `else if`s here and remove the `invalid` flag again, right?